frontend/src/dialogs/CreateClusterDialog.vue

```diff
@@ -582,7 +585,28 @@ export default {
       return get(this.shootDefinition, 'spec.addons', {})
     },
     regions () {
-      return this.regionsByCloudProfileName(this.cloudProfileName)
+      const { regionsWithSeed, regionsWithoutSeed } = this.regionsByCloudProfileName(this.cloudProfileName)
```

Comment:
rename regions to regionItems

create computed property

select default from regionsWithSeed